internal/pkg/daemon/profilerecorder/profilerecorder.go

```diff
@@ -423,9 +433,28 @@ func (r *RecorderReconciler) collectLogProfiles(
 func (r *RecorderReconciler) collectLogSeccompProfile(
 	ctx context.Context,
 	enricherClient enricherapi.EnricherClient,
+	parsedProfileName *parsedAnnotation,
 	profileNamespacedName types.NamespacedName,
 	profileID string,
 ) error {
+	labels, err := profileLabels(
```

Comment:
there's a bit of code duplication between the three recording functions, something to clean up perhaps